src/schema/notifications.ts

```diff
@@ -368,9 +367,9 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
           notificationsPageGenerator.nodeToCursor(page, args, node, index),
         (builder) => {
           builder.queryBuilder
-            .andWhere(`${builder.alias}."userId" = :user`, { user: ctx.userId })
-            .andWhere(`${builder.alias}."public" = true`)
-            .addOrderBy(`${builder.alias}."createdAt"`, 'DESC');
+            .andWhere(`un."userId" = :user`, { user: ctx.userId })
```

Comment:
Does this scale?
Just wondering if it doesn't break

Reply:
I did everything I could to make it fast. I believe it should be ok.
Basically pagination, ordering, filtering is done on a single table `UserNotification`. It should reduce the load significantly